applications/ParticleMechanicsApplication/python_scripts/mpm_gid_output_process.py

```diff
@@ -127,12 +127,12 @@ def PrintOutput(self):
             label = self.printed_step_count
         
         # Write results to the initiated result file
-        self._write_mp_results(label)
+        self._write_mp_results(time)
```

Comment:
Please note that the user may have selected to print the step count instead of the time (so use the `label` variable instead)

Reply:
@jcotela I am actually following the style in the gid_output_process. The label is used only for file naming if the multifile_flag is true. However, in the mpm_gid_output_process I don't have the multifile_flag yet.

Reply:
Actually, since you are writting the output file yourself, it might make sense to always print the time in the result block, and use the label only for the filename, but that is besides the point.

Given the current implementation, I would just remove the lines setting the label variable, since it is dead code (you are not using it anywhere, and you mention you are not yet supporting its use anyway).

Reply:
@jcotela agree with you. I have removed it from mpm_gid_output as suggested. Please recheck it. Thanks :+1: